lib/matplotlib/tests/test_axes.py

```diff
@@ -9747,3 +9747,8 @@ def test_pie_non_finite_values():
 
     with pytest.raises(ValueError, match='Wedge sizes must be finite numbers'):
         ax.pie(df, labels=['A', 'B', 'C'])
+
+def test_pie_all_zeros():
+    fig, ax = plt.subplots()
+    with pytest.raises(ValueError, match="All wedge sizes are zero"):
+        ax.pie([0,0], labels=["A","B"])
```

Comment:
```suggestion
        ax.pie([0, 0], labels=["A", "B"])
```
Not sure why the linter do not complain about this, but as you are fixing the errors, please fix this as well.